src/transformers/models/deformable_detr/image_processing_deformable_detr.py

```diff
@@ -1353,7 +1355,8 @@ def post_process_object_detection(
                 )
 
         prob = out_logits.sigmoid()
-        topk_values, topk_indexes = torch.topk(prob.view(out_logits.shape[0], -1), 100, dim=1)
+        k_value = min(top_k, prob.view(out_logits.shape[0], -1).size(1))
+        topk_values, topk_indexes = torch.topk(prob.view(out_logits.shape[0], -1), k_value, dim=1)
```

Comment:
Same here (would just need to be added here and then would be copied across when running `make fix-copies`

```suggestion
        prob = prob.view(out_logits.shape[0], -1)
        k_value = min(top_k, prob.size(1))
        topk_values, topk_indexes = torch.topk(prob, k_value, dim=1)
```